docs/guides/qedma-qesem.ipynb

```diff
@@ -229,25 +229,35 @@
     "\n",
     "| Option |  Choices | Description | Default |\n",
     "| -----| -----------| -------- | ------- |\n",
-    "| `estimate_time_only` |  True / False  | Specifies whether to execute the mitigation process or only get QPU time estimation. Please note that the QPU time estimation process does not consume QPU time.| False|\n",
+    "| `estimate_time_only` |  `\"analytical\"`  / `\"empirical\"`  | When set, the QESEM job will only calculate the QPU time estimation. Please see description below for more details.| |\n",
```

Comment:
@assafb, what is the default? `analytical`?

Reply:
@pandasa123  when a user set this flag then a value is mandatory.